README.md

```diff
@@ -200,14 +200,12 @@ the `tpm2_checkquote` utility in your path.
 
 ###### TPM 2.0 Resource Manager
 
-Note that it is recommended that you use the tpm2-abrmd resource manager
-(available at https://github.com/tpm2-software/tpm2-abrmd) as well instead of
-communicating directly with the TPM.  See README.md at that project for
-detailed instructions on how to build and install.
+Note that it is recommended that you use the tpm2-abrmd resource manager at [tpm2-abrmd](https://github.com/tpm2-software/tpm2-abrmd) as well instead of communicating directly with the TPM.  
```

Comment:
This is similar to the above: There's no reason to say "tpm2-abrmd" twice and only have the 2nd one be the link. Just use something like this:

```
Note that it is recommended that you use the [tpm2-abrmd](https://github.com/tpm2-software/tpm2-abrmd) resource manager as well instead of communicating directly with the TPM. 
```